styles.css

```diff
@@ -0,0 +1,101 @@
+
+/* CSS stylesheet */

```

Comment:
- [ ] Please remove any blank lines or indentations from your code.
Kindly keep in mind that we only add blank lines or indentation to separate two logical pieces of code or two large sections of code apart. This way, the code will remain clean, and the website will be more optimized.